public/userInfo.php

```diff
@@ -298,7 +298,7 @@ function resize() {
                 $userRank = $userMassData['Rank'];
                 $rankPct = sprintf("%1.2f", ($userRank / $countRankedUsers) * 100.0);
                 $rankOffset = (int) (($userRank - 1) / 25) * 25;
-                echo "<a href='/globalRanking.php?s=5&t=2&o=$rankOffset'>$userRank</a> / $countRankedUsers ranked users (Top $rankPct%)";
+                echo "<a href='/globalRanking.php?s=5&t=2&o=$rankOffset'>" . localized_number($userRank) . "</a> / " . localized_number($countRankedUsers) . " ranked users (Top $rankPct%)";
```

Comment:
This doesn't match the user cards logic for rendering site rank. See `buildCardRankData()` in _UserCard.php_.

In the user card, the rank is prefixed with a "#", and there is conditional logic for whether the Top % copy should display.

Reply:
Added "#" to the link, implemented the Top % hiding feature from UserCard.php